test/statetest/statetest_runner.cpp

```diff
@@ -18,6 +18,11 @@ inline bytes rlp_encode(const Log& log)
 
 namespace evmone::test
 {
+hash256 logs_hash(const std::vector<state::Log>& logs)
```

Comment:
Why dup here?

Reply:
Mistake. It was my first attempted location.